src/components/WizardModal/_wizard-modal.scss

```
@@ -0,0 +1,55 @@
+@import '../../globals/vars';
+
+.#{$iot-prefix}--wizard-modal {
+  .bx--modal-container {
```

These will need to use the configurable prefix from Carbon. I think it's just `prefix`?
```suggestion
  .#{prefix}--modal-container {
```